tests/queries/0_stateless/03008_s3_plain_rewritable_alter_partition.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
-# Tags: no-fasttest, no-shared-merge-tree
+# Tags: no-fasttest, no-shared-merge-tree, no-distributed-cache
 # Tag no-shared-merge-tree: does not support replication
+# Tag no-distributed-cache: requires investigation
```

Comment:
Is there a separate issue for investigation then?

Reply:
I created one: https://github.com/ClickHouse/clickhouse-private/issues/28599